ceph/ceph_admin.py

```diff
@@ -548,8 +548,55 @@ def ceph_iscsi(self, nodes, **kwargs):
             nodes: iscsi node list
             kwargs: key-value arguments
         """
-        # todo: Handle iscsi
-        raise NotImplementedError
+        op = kwargs.get('op', 'create').lower()
+
+        nodes = nodes if isinstance(nodes, list) else [nodes]
+        if op == "create":
+            iscsis = []
+            daemons = []
+            for node in nodes:
+                logger.info("adding iscsi on %s-%s" % (node.hostname,
+                                                       node.ip_address))
+                iscsis.append("{host}=iscsi.{host}".format(host=node.hostname))
+                daemons.append("iscsi.{host}".format(host=node.hostname))
+
+            if iscsis:
+                pool_name = "iscsi"
+
```

Comment:
Ignore if it is unreasonable...
```
script = f"""
ceph osd pool create {pool_name} 3 3 replicated
ceph osd pool application enable {pool_name} rbd
ceph orch apply iscsi {pool_name} user password --placement '{len(iscsis)};{";".join(iscsis)}'
"""[1:-1]

for cmd in script.splitlines(): self.shell(remote=self.installer, args=cmd)
```

Reply:
yes, its for future use, in case user deploy services with existing pool and may have further CLI statements.